### DISCUSSION TITLE
question about the auto formatting

### DISCUSSION THREAD

**Discussion by lephtHanded** [TriliumNext/Notes#418] [General]
Hey all, 
I use this awesome app for work and home.
I'm running in to a problem where I'm using a table to document a lot of search strings and the quote marks ["] are turning in to the fancy open and end quotes.  is there any way to make the quotes just the regular kind?  

when i copy and paste from Trilium, (even with paste as plain text) the fancy quotes are transferred over.

**Comment by eliandoran** [comment]
@lephtHanded , that is a very good question. 
Would you prefer to have an option to disable it globally or are you thinking of a paste function that does not convert the quotes?

**Reply by lephtHanded** [reply]
I think global option would be easier to implement.

Personally, I don't see the need for them at all - but if others do, maybe a per-page option?

**Reply by eliandoran** [reply]
@lephtHanded , OK, I will look into the possibility of implementing an option for it in #419 .
